.github/exercises/K-functions-parameters/exercise3.js

```diff
@@ -0,0 +1,8 @@
+// Write your function here
+function createGreeting(name){
```

Comment:
```
# #function createGreeting(a){
    return "hell, my name is " + a;
}

var greeting = createGreeting("Daniel");
```

Hi, please see the example above,  your variable will contain the callback function and there we assign the name, so your return must contain the rest of the sentence in string and then you can make the interpolation. I hope this helps to you. ☺️